integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/DebugRunFacet.java

```diff
@@ -22,25 +24,32 @@
  * integration and provide details on potential problems.
  */
 @Getter
+@JsonSerialize(using = DebugRunFacetSerializer.class)
 public class DebugRunFacet extends OpenLineage.DefaultRunFacet {
   private final ClasspathDebugFacet classpath;
   private final SystemDebugFacet system;
   private final SparkConfigDebugFacet config;
   private final LogicalPlanDebugFacet logicalPlan;
   private final MetricsDebugFacet metrics;
+  private final List<String> logs;
```

Comment:
I don't understand how you fill this field - should this not be very much dependent of logger config and Spark distribution?

Reply:
This will not contain slf4j logs. It can just contain some extra information about debug facet like: 
 - debug facet serialization size exceeds certain size
 - no inputs or no output dataset have been detected.